app/src/main/res/layout/item_attendee.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/cardView"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/cardView_height"
+    android:focusable="true"
+    android:foreground="?android:attr/selectableItemBackground">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+            android:id="@+id/tv_attendee"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/tv_margin"
+            android:padding="@dimen/tv_padding"
+            android:text="@string/attendees"
+            android:textSize="@dimen/tv_text_size_small"
+            app:layout_constraintEnd_toEndOf="@id/guide_70_percent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageButton
+            android:id="@+id/btn_delete"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/btn_margin"
+            android:background="@drawable/custom_button"
+            android:padding="10dp"
```

Comment:
can we move this to dimen.xml?

Reply:
Which part do you mean? I use dimen for all attributes

Reply:
I found one padding which was not defined to dimen and changed it

Reply:
yes there was only one in line 32